src/runner_manager.py

```diff
@@ -398,11 +401,14 @@ def _get_runner_github_info(self) -> Dict[str, SelfHostedRunner]:
         remote_runners_list: list[SelfHostedRunner] = []
         if isinstance(self.config.path, GitHubRepo):
             remote_runners_list = self._clients.github.actions.list_self_hosted_runners_for_repo(
-                owner=self.config.path.owner, repo=self.config.path.repo
+                owner=self.config.path.owner,
+                repo=self.config.path.repo,
+                per_page=100,
```

Comment:
Aren't we still limited to 100 runners? We're planning to go to 162 pretty soon

Reply:
The `per_page` has a 100 max size.
Should we migrate to PyGitHub?

Reply:
Is this just for logging or does it have a functional impact?

Reply:
Will update to use `paged` generator provided by ghapi to handle pagination.